odim-controller/helmcharts/odimra-config/templates/configmaps.yaml

```diff
@@ -129,6 +129,11 @@ data:
     		"UpdateService",
     		"TelemetryService"
     	],
-        "ConnectionMethodConf": {{ .Values.odimra.connectionMethodConf | toJson }},
-    	"SupportedPluginTypes": ["Compute", "Fabric", "Storage"]
+      "ConnectionMethodConf": {{ .Values.odimra.connectionMethodConf | toJson }},
+    	"SupportedPluginTypes": ["Compute", "Fabric", "Storage"],
+      "EventConf": {
+                 "DeliveryRetryAttempts" : 3,
+                 "DeliveryRetryIntervalSeconds" : 60,
+                 "RetentionEventsInMinutes" : 10
```

Comment:
Rename it to `RetentionOfUndeliveredEventsInMinutes`

Reply:
changed